lua/entities/gmod_wire_hologram.lua

```diff
@@ -120,14 +120,20 @@ if CLIENT then
 		end
 
 		self:SetupClipping()
-
+		
+		render.CullMode(self:GetNWBool("invert_model", 0))
```

Comment:
Can you store the result of `self:GetNWBool` as a variable so it isn't called twice?

Reply:
And if you make the change above would need to do `inverted and 1 or 0`, (or maybe not, if lua can coerce a boolean into a number in the `render.CullMode` C function, if you want to try that out)